test/mercury/unique-display-name.test.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+/* eslint-disable no-undef */
+"use strict";
+import {
+  getHappySpec,
+  renderSpecAsFile,
+  breaksOnlyOneRule,
+  conforms
+} from "../utils.test";
+import { validateFile } from "../../src/validator";
+
+describe("unique display name validation", () => {
+  const PROFILE = "mercury";
+  const RULE = "http://a.ml/vocabularies/data#unique-display-name";
+  let doc;
+
+  beforeEach(() => {
+    doc = getHappySpec();
+  });
+
+  it("should pass if all the display names are unique", async () => {
+    const result = await validateFile(renderSpecAsFile(doc), PROFILE);
+
+    conforms(result);
+  });
+
```

Comment:
Pls add some tests with case changes like "notUnique", "NOTUnique"

Reply:
If a display name is not camelcase `camelcase-method-displayname` should already fail that RAML.

Reply:
Sorry for the typo... something like "notUnique and notuNique" that passes other tests and fails/passes uniqueness rule..